package.json

```diff
@@ -27,11 +28,11 @@
     "extends @edx/browserslist-config"
   ],
   "dependencies": {
-    "@edx/brand": "npm:@edx/brand-openedx@1.2.0",
+    "@edx/brand": "npm:@edx/brand-edx.org@2.2.0-alpha.9",
```

Comment:
TODO: will need to make similar changes to `@edx/brand-openedx` so that the correct files exist in that NPM package, too.